libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/eh/RoadName.kt

```diff
@@ -1,54 +0,0 @@
-package com.mapbox.navigation.base.trip.model.eh
-
-/**
- * Road name information
- *
- * **NOTE**: The Mapbox Electronic Horizon feature of the Mapbox Navigation SDK is in public beta
- * and is subject to changes, including its pricing. Use of the feature is subject to the beta
- * product restrictions in the Mapbox Terms of Service.
- * Mapbox reserves the right to eliminate any free tier or free evaluation offers at any time and
- * require customers to place an order to purchase the Mapbox Electronic Horizon feature,
- * regardless of the level of use of the feature.
- *
- * @param name road name
- * @param shielded is the road shielded?
- */
-class RoadName internal constructor(
-    val name: String,
-    val shielded: Boolean
```

Comment:
What did this flag represent? How do developers get the same information now, is it `RoadComponent#shield != null`? Let's add this to the changelog entry as the migration instructions.

Reply:
This flag was a temporary solution when Shield class was not defined. Now the Shield itself can be used. For update please mind that `shielded` was just `shield != null` .